eras/conway/test-suite/cardano-ledger-conway-test.cabal

```
@@ -54,8 +54,7 @@ library
         cardano-strict-containers,
         microlens,
         tasty,
-        tasty-quickcheck,
-        QuickCheck
+        tasty-quickcheck
```

```suggestion
        tasty-quickcheck,
```
Maybe leave the trailing comma so diffs look nicer.

---

Oh, never mind, I forgot we auto-format this now

---

Yep auto-format :slightly_smiling_face: